corehq/apps/domain/static/domain/js/case-search-config.js

```diff
@@ -50,14 +50,33 @@ hqDefine('domain/js/case-search-config.js', function () {
             self.fuzzyProperties.push(new CaseTypeProps('', ['']));
         }
 
+        self.change = function(){
+            self.saveButton.fire('change');
+        };
+        self.fuzzyProperties.subscribe(self.change);
+
         self.addCaseType = function () {
             self.fuzzyProperties.push(new CaseTypeProps('', ['']));
+            self.change();
         };
         self.removeCaseType = function (caseType) {
             self.fuzzyProperties.remove(caseType);
+            self.change();
```

Comment:
doesn't the SaveButton listen to all changes on `input` elements and then fire the change event for you? i would've expected it to just work